File: 2107-find-unique-binary-string/find-unique-binary-string.py
class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        n = len(nums)
        ans = ""

        for i in range(n):
            if nums[i][i] == "0":
                ans += "1"
            else:
                ans += "0"

        return ans

File: 2107-find-unique-binary-string/find-unique-binary-string.cs
public class Solution {
    public string FindDifferentBinaryString(string[] nums) {
        int n = nums.Length;
        StringBuilder ans = new StringBuilder();

        for (int i = 0; i < n; i++) {
            if (nums[i][i] == '1') {
                ans.Append('0');
            }
            else {
                ans.Append('1');
            }
        }

        return ans.ToString();
    }
}